spec/api/comment_thread_spec.rb

```diff
@@ -446,20 +391,20 @@ def test_paged_order (sort_spec, expected_order, filter_spec=[], user_id=nil)
               make_comment(@threads["t5"].author, @threads["t5"], "extra comment")
               @threads["t7"].pinned = true
               @threads["t7"].save!
-              expected_order = move_to_front(move_to_end(@default_order, "t5"), "t7")
```

Comment:
Sort descending means most recent first (like in the webapp).

This was sorting ascending, which meant most recent last.  Since the first line is to make a new comment ("t5"), which would be most recent, and we were sorting ascending, move_to_end would just make it last.  We now want it to be first, except for the pinned item.

I guess we might want to pin t7 before making t5, to prove t7 goes to the front because it is pinned, and not because it has been edited more recently.  But, I did just test this and it works either way.  You ok leaving this?